experimental/GaloisGrp/GaloisGrp.jl

```diff
@@ -361,7 +383,7 @@ function probable_orbit(G::Oscar.PermGroup, f::SLPoly)
 end
 
 #TODO:
-#- ansehen der ZykelTypen um Sn/An zu erkennen
+#- ansehen der ZykelTypen um Sn/An zu erkennen (as well as to avoid 2-sum)
```

Comment:
See also https://github.com/gap-system/gap/blob/82afa0cc35b594eff57d1e6aecb0142dca23c244/lib/gpprmsya.gi#L555 resp. "Ákos Seress, Permutation group algorithms. Cambridge Tracts in Mathematics, 152. Section 10.2"

If you know the action is transitive and there are cycles of prime length p with n/2 < p < n - 2 then the group contains A_n. The sign of the generators of course tell you whether it is A_n or S_n (cycle types of course also give you this)